Kuzzle.Tests/Utils/QueryUtils.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Newtonsoft.Json.Linq;
+
+namespace Kuzzle.Tests.Utils {
```

Comment:
Too easy to have a name collision, with tests unexpectedly using the KuzzleSdk.Utils domain instead of this one.